.github/workflows/ansible-test.yml

```diff
@@ -0,0 +1,124 @@
+# README FIRST
+# 1. If you don't have unit tests remove that section
+# 2. If your collection depends on other collections ensure they are installed, see "Install collection dependencies"
+# If you need help please ask in #ansible-devel on Freenode IRC
```

Comment:
@gundalow just checking, is #ansible-devel accurate ?

Reply:
#ansible-devel is correct